app/main.py

```diff
@@ -1,13 +1,17 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # white your code here
-    pass
+    return {"errors": [format_linter_error(i) for i in errors],
+            "path": file_path,
+            "status": "passed" if errors == [] else "failed"}
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # white your code here
-    pass
+    return [format_single_linter_file(key, value)
+            for key, value in linter_report.items()]
```

Comment:
key and value are uninformative names for variables

Reply:
What values are considered informative?

Reply:
path and error